objectbox/lib/src/annotations.dart

```diff
@@ -1,27 +1,57 @@
+/// Entity annotation is used to on a class to let ObjectBox know it should
+/// store it - making the class a "persistable Entity".
+///
+/// This annotation is matched by ObjectBox code generator when you call
+/// `pub run_build_runner build`. The generator creates `objectbox.g.dart` with
+/// all the binding code necessary to store the class in the database.
 class Entity {
+  /// ObjectBox keeps track of entities and properties by assigning them unique
+  /// identifiers, UIDs, during the code-generation phase. All those UIDs are
+  /// stored in a file objectbox-model.json in your package and are looked up by
+  /// name. If the name changes a new uid is assigned, effectively creating a
+  /// new Entity in the database.
+  ///
+  /// If you explicitly specify a [uid] on an [Entity], ObjectBox would be able
+  /// to identify it even after you change the name and would update the
+  /// database accordingly on the next application launch - renaming the stored
+  /// Entity instead of creating a new one.
   final int /*?*/ uid;
 
+  /// Create an Entity annotation.
   const Entity({this.uid});
 }
 
-/// A dart int value can map to different OBXPropertyTypes,
-/// e.g. Short (Int16), Int (Int32), Long (Int64), all signed values.
-/// Also a dart double can also map to e.g. Float and Double
-///
-/// Property allows the mapping to be specific. The defaults are
-/// e.g. Int -> Int64, double -> Float64, bool -> Bool.
+/// Property annotation enables you to explicitly specify configure some details
+/// about how a field is stored in the database.
 ///
-/// Use OBXPropertyType and OBXPropertyFlag values, resp. for type and flag.
+/// Use PropertyType and OBXPropertyFlag values, resp. for type and flag.
```

Comment:
```suggestion
/// Use PropertyType values for type.
```
Or remove altogether?

Reply:
yes, removed